applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -541,8 +541,10 @@ gateway_to_endpoint(Number
          ,{<<"Endpoint-Options">>, EndpointOptions}
          ,{<<"Endpoint-Progress-Timeout">>, wh_util:to_binary(ProgressTimeout)}
          ,{<<"Custom-Channel-Vars">>, wh_json:from_list(CCVs)}
-         | maybe_get_t38(Gateway, JObj)
-        ])).
+        ]
+        ++ maybe_get_t38(Gateway, JObj)
+        ++ wh_json:copy(<<"Outbound-Caller-ID-Number">>, JObj)
+        ++ wh_json:copy(<<"Outbound-Caller-ID-Name">>, JObj))).
```

Comment:
or you could just set
{<<"Outbound-Caller-ID-Name">>, wh_json:get_value(<<"Outbound-Caller-ID-Name">>, JObj}
within the proplist above